go.mod

```diff
@@ -3,78 +3,79 @@ module github.com/longhorn/longhorn-manager
 go 1.13
 
 replace (
-	k8s.io/api => k8s.io/api v0.0.0-20191004102349-159aefb8556b
-	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20191004074956-c5d2f014d689
-	k8s.io/client-go => k8s.io/client-go v11.0.1-0.20191004102930-01520b8320fc+incompatible
+	k8s.io/api => k8s.io/api v0.16.15
```

Comment:
Is this the result after `go mod tidy`? Also, do we need ping so many packages here? Most of them don't look like direct dependencies.

Reply:
> Is this the result after go mod tidy?
Yes, Kubernetes team tags the release to make it easier for the users to consume. e.g not having to read the longhorn hascode.

> Also, do we need ping so many packages here? Most of them don't look like direct dependencies.

Which one is not direct dependency?